lib/main.dart

```diff
@@ -82,6 +83,7 @@ class LinShareApp extends StatelessWidget {
           // the app on. For desktop platforms, the controls will be smaller and
           // closer together (more dense) than on mobile platforms.
           visualDensity: VisualDensity.adaptivePlatformDensity,
+          unselectedWidgetColor: AppColor.unselectedElementColor
```

Comment:
Sr, can you share a bit about this? I don't understand what it is used for.

Reply:
This will colors all unselected widget like checkbox, toggle, radio etc 

> The color used for widgets in their inactive (but enabled) state. For example, an unchecked checkbox. Usually contrasted with the accentColor. See also disabledColor.

In Figma, this is the color of all these items

Reply:
cool